docs/symphony-book/get-started/deploy_solution_to_azure_iot_edge.md

```diff
@@ -73,20 +76,84 @@ You need to prepare a Linux virtual machine or physical device for IoT Edge. In
   sudo iotedge config mp --connection-string '<REPLACE_WITH_DEVICE_CONNECTION_STRING>'
   ```
 
+**Note**
+If you encounter below package dependency error during installing `aziot-edge`:
+```
+The following packages have unmet dependencies:
+ aziot-identity-service : Depends: libssl1.1 (>= 1.1.1) but it is not installable
+```
+
+This is because Ubuntu 22.04 uses `libssl3` and `libssl1.1` is deprecated on it. To address the issue, check the latest versions for Edge and IoT Identity Service and install them from the release page.
+
+```shell
+# check the latest aziot-edge version
+wget -qO- https://raw.githubusercontent.com/Azure/azure-iotedge/main/product-versions.json | jq -r '
+  .channels[]
+  | select(.name == "stable").products[]
+  | select(.id == "aziot-edge").components[]
+  | select(.name == "aziot-edge").version
+'
+# check the latest aziot-identity-service version
+wget -qO- https://raw.githubusercontent.com/Azure/azure-iotedge/main/product-versions.json | jq -r '
+  .channels[]
+  | select(.name == "stable").products[]
+  | select(.id == "aziot-edge").components[]
+  | select(.name == "aziot-identity-service").version
+'
+
+# download and install 
+wget https://github.com/Azure/azure-iotedge/releases/download/<AZIOT_EDGE_VERSION>/aziot-identity-service_<AZIOT_IDENTITY_SERVICE_VERSION>-1_ubuntu22.04_amd64.deb -O aziot-identity-service.deb
+sudo dpkg -i aziot-identity-service.deb
+
+wget https://github.com/Azure/azure-iotedge/releases/download/<AZIOT_EDGE_VERSION>/aziot-edge_<AZIOT_EDGE_VERSION>-1_ubuntu22.04_amd64.deb -O aziot-edge.deb
+sudo dpkg -i aziot-edge.deb
+```
+
+### 3. Deploy modules on your IoT Edge device
+
+To deploy your IoT Edge modules, go to your IoT hub in the Azure portal, then:
+
+1. Select Devices from the IoT Hub menu.
+
+2. Select your device to open its page.
+
+3. Select the Set Modules tab.
+
+4. Select Review + create at the bottom.
+
+5. Select Create to deploy the modules.
+
+When the module setup completes, you should see two modules in the list: `$edgeAgent` and `$edgeHub`.
+For more information, please refer to [Create and provision an IoT Edge device on Linux using symmetric keys](https://learn.microsoft.com/en-us/azure/iot-edge/how-to-provision-single-device-linux-symmetric?view=iotedge-1.5&tabs=azure-cli%2Cubuntu).
+
 ## OPTION 1: Use Maestro
 
 To use this option, first install Maestro and the Symphony API. For more information, see [Use Symphony with the Maestro CLI tool](./quick_start_maestro.md).
 
 Once you have maestro installed, you can launch this sample with the following command:
 
 ```bash
-maestro samples run hello-iot-edge --set iot-hub-key=<REPLACE_WITH_HUB_KEY> --set iot-hub-name=<REPLACE_WITH_HUB_NAME> --set device-name=s8c-vm
+maestro samples run hello-iot-edge --set iot-hub-key=<REPLACE_WITH_HUB_KEY> --set iot-hub-name=<REPLACE_WITH_HOST_NAME> --set device-name=s8c-vm
```

Comment:
IoT Hub name and IoT Hub Host name are two different things. The IoT Hub name is the name of the hub, and the host name is the fully qualified hub name. Please double check the right term is used and hereafter.

Reply:
Yes, I have tested locally with the host name. It seems that in the upgraded iot-edge package version, the host name is used in the URL retrieving module twin.